user_guide/intro_navigation.rst

```diff
@@ -0,0 +1,110 @@
+.. _user-guide-getting-started-controls:
+  
+Navigation Basics
+=================
+
+OroCRM provides a broad set of navigation tools that allow you to easily access necessary information and tools. The 
+system helps to ensure that the frequent actions can be performed with minimal effort by keeping required information at 
+hand. This means no important updates are missed.
+
+This section provides an overview of the navigation elements in OroCRM.
+
+
+.. _user-guide-navigation-menu:
+
+The Menu Bar
+------------
+
+The menu bar is the main mode of accessing data in OroCRM. It is always displayed at every page of OroCRM. Subject to 
+your organization configuration, it may be displayed horizontally or vertically. In the latter case, the menu items will 
+be displayed as icons.
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_top.png
+
+|
+
+.. image:: ./img/navigation/menu/nav_bar_side.png
+
+|
+
+In the mobile version the menu bar can be accessed via the "hamburger" button next to the organization name:
+
+|
+    
+.. image:: ./img/navigation/menu/header_mobile.png
+
+|
+
+The list of menu items available to you, depends on the access and permission settings defined in the organization.
+
+Hover the mouse over a menu item to expand the drop-down menu with the second-level items. Click on them to access 
+corresponding pages or grids. (See :ref:`Data management basics <user-guide-data-management-basics>` for further 
+details.)
+  
+  
+.. _user-guide-navigation-panel:
+
+Navigation Tools
+----------------
+
+At the very top of the page, there is navigation panel that contains different navigation tools, as shown below.
```

Comment:
Edit: At the very top of the page, there is a navigation panel that contains different navigation tools, as shown below.